### ISSUE TITLE
Disabling virtual stick mode on mobile disconnect.

### ISSUE THREAD

**Issue by Abhishek284** [dji-sdk/Mobile-SDK-Android#272]
Platform: Android
SDK: 4.4.1

I am enabling virtual stick mode for doing some action and disabling it as soon as its done. But if I disconnect the mobile before disabling it, I will not be able to use the remote controller sticks and aircraft hovers. 

Is there a way to disable virtual stick mode once mobile is disconnected?

**Comment by wocoltd** [comment]
I agree that some kind of time-out would be helpful in this situation. Although you can regain control with the sport mode switch, in some cases we want to disable that in our app, which would leave us with no way of regaining control?

**Comment by Djacob502** [comment]
I think I'm missing something - why would you want this behavior? For us - if the mobile disconnects then an appropriate solution is to show a message to the user to switch the mode switch as wocoltd describes. I admit that this is a training issue and pilots need to be trained not to panic and do this behavior. I"m more concerned that if this default behavior is moved to the firmware/drone then we will have to deal with control returning to the sticks when we do not want it.
